### PR TITLE
Add Ruby Gems viewer package

### PR DESCRIPTION
**Pull request by ZainIftikhar7vals**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is Ruby Gem viewer.

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by kaste** [comment]
This looks good to me but I do wonder

```
      gem_home = os.popen("echo $GEM_HOME").read().split("\n")[0]
      gems_directories = [
        os.path.expanduser("%s/gems/" % gem_home),
        os.path.expanduser("%s/bundler/gems/" % gem_home)
      ]
```

Isn't that readable just from within Sublime Text using `os.environ["GEM_HOME"]`.  Would be nice to make it available for Windows too.  Is that impossible to achieve?

The latest tag right now is "0.2.0-alpha".  Not sure if a suffix (`-alpha`) counts as release version.  braver will know that, or change it to just `0.3.0`.

**Comment by kaste** [comment]
* Please lift the plugin to run on the 3.8 host. For that, create a file .python-version with its only contents 3.8\n in the root of your package.

**Comment by ZainIftikhar7vals** [comment]
> * Please lift the plugin to run on the 3.8 host. For that, create a file .python-version with its only contents 3.8\n in the root of your package.

Can you please explain it a little bit more, as I am new to Sublime package development?

**Comment by ZainIftikhar7vals** [comment]
> Would be nice to make it available for Windows too. Is that impossible to achieve?

It's definitely possible, but since I don't have access to a Windows machine right now, I can't test it. However, I plan to do so in the future.

**Comment by kaste** [comment]
Yeah, to "lift" the package (or better worded: choose the environment) create a file just like https://github.com/kaste/InlineOutline/blob/master/.python-version.

**Comment by ZainIftikhar7vals** [comment]
@kaste All comments resolved.

**Comment by kaste** [comment]
We need @braver for the merge.

File: repository/g.json [hunk 1]
```diff
@@ -1920,6 +1920,19 @@
 				}
 			]
 		},
+		{
+			"name": "GotoRubyGem",
+			"details": "https://github.com/ZainIftikhar7vals/GotoRubyGem",
+			"readme": "https://raw.githubusercontent.com/ZainIftikhar7vals/GotoRubyGem/main/README.md",
```
Comment (koen-at-peppered) [inline repository/g.json:1926]: Your readme is in the default location so no need to reference it here.
```suggestion
```

File: repository/g.json [hunk 2]
```diff
@@ -1920,6 +1920,19 @@
...
+			"labels": ["code navigation", "ruby gems"],
```
Comment (koen-at-peppered) [inline repository/g.json:1927]:
```suggestion
			"labels": ["code navigation", "ruby gems", "ruby"],
```
A general "ruby" label will be more useful actually.

**Review by koen-at-peppered — CHANGES_REQUESTED** [review]
Just some final tweaks to the entry here, looks good otherwise 👍

**Comment by ZainIftikhar7vals** [comment]
@koen-at-peppered changes have been incorporated, please review.

**Pull request merged**